aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_metric_attribute_generator.py

```diff
@@ -344,10 +348,18 @@ def _set_remote_type_and_identifier(span: ReadableSpan, attributes: BoundedAttri
     remote_resource_type: Optional[str] = None
     remote_resource_identifier: Optional[str] = None
 
-    # Only extract the table name when _AWS_TABLE_NAMES has size equals to one
-    if is_key_present(span, _AWS_TABLE_NAMES) and len(span.attributes.get(_AWS_TABLE_NAMES)) == 1:
-        remote_resource_type = _NORMALIZED_DYNAMO_DB_SERVICE_NAME + "::Table"
-        remote_resource_identifier = span.attributes.get(_AWS_TABLE_NAMES)[0]
+    # Only extract remote_resource_type and remote_resource_identifier
```

Comment:
I think we should consider tying the implementation of this method to `_set_remote_service_and_operation`. E.g. Change `_set_remote_service_and_operation` to `_set_remote_attributes` and have it set remote service, operation, resource type, and resource identifier all in one go. All attributes are tied together, so it makes sense to have the logic be centralized.